plugins/woocommerce/tests/e2e-pw/tests/shopper/order-email-receiving.spec.js

```diff
@@ -1,5 +1,5 @@
 const { test, expect } = require( '@playwright/test' );
-const { customerDetails, storeDetails } = require( '../../test-data/data' );
+const { customer, storeDetails } = require( '../../test-data/data' );
```

Comment:
Using the `customer` object from `data.js` as mentioned [here](https://github.com/woocommerce/woocommerce/pull/35492/files#r1013814136).